README.md

```diff
@@ -25,6 +25,7 @@ libraryDependencies += "org.portable-scala" %%% "portable-scala-reflect" % "1.0.
 * Scala 2.11.x, 2.12.x and 2.13.x
 * Scala/JVM
 * Scala.js 0.6.x and 1.x
+* Scala Native 0.4.0
```

Comment:
```suggestion
* Scala Native 0.4.x
```
Could this be? /cc @sjrd

Reply:
Yes, I can change that, so that it is consistent with other platforms.
By my understanding, any patch level change (.x) which broke
backward compatibility would trigger at least a minor version change.

The explicit 0.4.0 comes from my long standing reluctance to document
something that I have not seen work with my own eyes.  I can get over that
here.

thanks.

L

Reply:
Changed, thank you.